app/src/main/res/anim/fadein.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<alpha xmlns:android="http://schemas.android.com/apk/res/android"
+    android:duration="2000"
```

Comment:
??? 2 seconds?!

Reply:
adjusted to 5 seconds, was testing with faster speeds to make it didn't persist very long